po/ru.po

```diff
@@ -733,27 +733,26 @@ msgstr "Не выбрано ни одной ленты!"
 #. "Sort by (f)irsttag/..." and "Reverse Sort by
 #. (f)irsttag/..." messages
 #: src/feedlistformaction.cpp:130 src/feedlistformaction.cpp:173
-#, fuzzy
 msgid "ftaulsn"
-msgstr "ftauln"
+msgstr "ftaulsn"
 
 #: src/feedlistformaction.cpp:132
-#, fuzzy
 msgid ""
 "Sort by (f)irsttag/(t)itle/(a)rticlecount/(u)nreadarticlecount/(l)astupdated/"
 "late(s)tunread/(n)one?"
 msgstr ""
 "Сортировать по первой метке (f)/заголовку (t)/количеству записей (a)/"
-"непрочитанным (u)/времени последнего обновления (l)/никак (n)?"
+"непрочитанным (u)/времени последнего обновления (l)/(s)времени последнего "
```

Comment:
The other choices have the key behind the text.
Is it intentional to put the `(s)` in front here?